agent/xds/endpoints.go

```diff
@@ -308,6 +377,8 @@ func makeLoadAssignmentEndpointGroup(
 		}, true
 	}
 
+	// TODO(rb): shouldn't we omit gateways that are unhealthy?
```

Comment:
I'd say no - if we just tell Envoy they are unhealthy it can make more sensible choices like enforce it's panic threshold if more than 50% are healthy or similar. Saves us from building that kind of logic in here...